internal/ssh/flags.go

```diff
@@ -111,3 +133,28 @@ func (d *durationFlagValue) Get() any {
 func (d *durationFlagValue) String() string {
 	return (*time.Duration)(d).String()
 }
+
+// stringFlagValue is a wrapper around string that implements the flag.Value interface using a custom parser.
+type stringFlagValue string
+
+// addStringFlag adds a flag for a string to the given flag.FlagSet.
+func addStringFlag(fs *flag.FlagSet, flagValue *string, name string, value string, usage string) {
+	*flagValue = value
+	fs.Var((*stringFlagValue)(flagValue), name, usage)
+}
+
+// Set implements the flag.Value interface.
+func (sf *stringFlagValue) Set(s string) error {
+	*sf = stringFlagValue(s)
+	return nil
+}
+
+// Get implements the flag.Getter interface.
+func (sf *stringFlagValue) Get() any {
+	return string(*sf)
+}
+
+// String implements the flag.Value interface.
+func (sf *stringFlagValue) String() string {
+	return string(*sf)
+}
```

Comment:
You shouldn't need all this, [`flag.StringVar`](https://pkg.go.dev/flag#StringVar) can do it already.

The reason why I have a custom duration parser above is because go's default duration parser doesn't support days 😄

Reply:
Ah, it introduced some weirdness and didn't like it - I will investigate more though and see what went wrong there.